### PR TITLE
Fixes for #131 + #132 (MSVC related)

### PR DESCRIPTION
**Pull request by karel-m**
see #131 and #132

### CODE REVIEW

File: bn_mp_set_double.c [hunk 1]
```diff
@@ -49,7 +49,7 @@ int mp_set_double(mp_int *a, double b)
    return MP_OKAY;
 }
 #else
-#  warning "mp_set_double implementation is only available on platforms with IEEE754 floating point format"
+#pragma message("mp_set_double implementation is only available on platforms with IEEE754 floating point format")
```
[the remark this answers is not in the document]
Comment (karel-m) [reply bn_mp_set_double.c]: Perhaps @minad or @czurnieden may have an answer (or at least an opinion).

I agree with anything that does not end up with a compiler error on MSVC.

**Comment by minad** [comment]
> Or make a preprocessor branch and use both `#warning` and `#pragma message` according to the compiler.

:+1: to that

**Comment by czurnieden** [comment]
Added preprocessor branch because `#pragma message()` lacks support in several compilers. Please check if MSVC eats it.

Because of that little hiccup a couple of days ago, I pushed it to my own repository at github to avoid more mess. All are gathered now at the pull request  #113 . Sorry for the inconvenience.

@minad There is still a significant lack of documentation in `bn.tex` regarding the functions in `bn_mp_set_double.c`, `bn_mp_get_double.c`, and the `bn_mp_tc_*c` ones. Please add some this weekend or I'll have to do it and that'll make me even grumpier than I already am.

Why this haste you might ask? Debian needs the new version ASAP, see #122 for more information.

**Comment by sjaeckel** [comment]
> those PR's shouldn't have been merged in the first place, as the documentation was missing.

@karel-m can we probably add tooling to prevent e.g. travis from turning green if there are new API functions added but the docs are missing?

**Comment by sjaeckel** [comment]
@karel-m does this build fine on your msvc variants now?

**Comment by sjaeckel** [comment]
please merge when you think it's ready :)

**Pull request merged**